src/test/java/org/traccar/protocol/WialonProtocolDecoderTest.java

```diff
@@ -28,10 +30,10 @@ public void testDecode() throws Exception {
 
         verifyNull(decoder, text(
                 "#L#123456789012345;test"));
-        
```

Comment:
Don't forget to undo all these changes and clean up your PR. Let me know once it's ready for another review.